packages.dhall

```diff
@@ -0,0 +1,6 @@
+let upstream =
+      https://raw.githubusercontent.com/purescript/package-sets/prepare-0.15/src/packages.dhall
+        sha256:0c9c7cd174b7529f2862d52fbfee8cffe1ba8b9cfed3f0f600bdb5be0a14c0ff
```

Comment:
Note: this hash shouldn't be included for the time being as the package set's contents will change until we have a  released version...